docs/Reference/Pantheon-API-Methods.md

```diff
@@ -640,19 +660,27 @@ None
     }
     ```
 
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{gasPrice}"}' http://localhost:8547/graphql
+    ```
+    
+    ```json tab="GraphQL result"
+    {
+      "data" : {
+        "gasPrice" : "0x3e8"
+      }
+    }
+    ```   
+
 ### eth_accounts
 
 Returns a list of account addresses that the client owns.
 
 !!!note
-<<<<<<< HEAD:docs/Reference/JSON-RPC-API-Methods.md
```

Comment:
This merge artifact needs to be resolved. I'm surprised git didn't complain about it when submitting.

Reply:
It is gone now

Reply:
I've raised https://pegasys1.atlassian.net/secure/RapidBoard.jspa?rapidView=38&view=planning&selectedIssue=PAN-2716 to add the pretty printed tab